WebHostLib/api/downloads.py

```diff
@@ -0,0 +1,48 @@
+from flask import redirect, Response
+from pony.orm import commit, db_session, select
+
+from Utils import __version__
+from WebHostLib import cache
+from WebHostLib.models import ArchipelagoDownload
+from . import api_endpoints
+
+
+@cache.cached
+@api_endpoints.route("/downloads/<string:build>")
+def get_download(build: str) -> Response:
+    download_data = select(download for download in ArchipelagoDownload
+                           if download.platform == build).first()
+    if download_data and download_data.version >= __version__:
```

Comment:
I think this logic and comparison are flawed:
* if a new version is found, it'll never check again
* download_data.version is a string, `__version__` is a tuple
* if a new version was not found, it'll always hit the github endpoint

Reply:
i did this to prevent it from checking for updates constantly... should i remove that and just go back to checking every 5 minutes?

Reply:
I think so.
This would ignore critical updates (say 0.6.1) that only affects clients while not needing a webhost update (can stay on 0.6.0).